app.py

```diff
@@ -1,12 +1,40 @@
 from flask import Flask, render_template, request, redirect, url_for
 import session_items as session
+from operator import itemgetter
 
 app = Flask(__name__)
 app.config.from_object('flask_config.Config')
 
-@app.route('/')
-def index():
-    return 'Hello World!'
+
+@app.route('/', methods=['GET'])
+def get_todo_list():
+    todo_list = session.get_items()
+    newlist = sorted(todo_list, key=itemgetter('status'), reverse=True)
+    return render_template('index.html', items=newlist)
+
+
+@app.route('/create', methods=['POST'])
+def create():
+    session.add_item(request.form['add_todo'])
+    return redirect('/')
+
+
+@app.route('/update', methods=['POST'])
+def update():
+    for id in request.form:
```

Comment:
This approach limits you to having one input field per todo. You might want to move the `<form>` element into the jinja for loop in `index.html` to make this more extensible in future.